corehq/apps/userreports/reports/specs.py

```diff
@@ -119,7 +119,7 @@ def get_sql_column_config(self, data_source_config, lang):
                 header=self.get_header(lang),
                 agg_column=SQLAGG_COLUMN_MAP[self.aggregation](self.field, alias=self.column_id),
                 sortable=False,
-                data_slug=self.column_id,
+                slug=self.column_id,
```

Comment:
This line was unrelated to the bug, but this keyword arg was wrong. I don't *think* that changing this line will change anything, but we should definitely wait for the tests.